client/me/profile/index.jsx

```diff
@@ -124,6 +126,23 @@ class Profile extends Component {
 							</FormSettingExplanation>
 						</FormFieldset>
 
+						<FormFieldset>
+							<div className="form-label">{ this.props.translate( 'Public profile' ) }</div>
+							<FormSettingExplanation>
+								<span>
+									{ this.props.translate(
+										'A list of your public posts and public lists can be found at {{a}}{{url/}}{{/a}}',
```

Comment:
I think it is beneficial to have some general content that don't require any update if we decide to introduce a new user profile tab in the future.

Reply:
@mehmoodak Can you elaborate? Do you mean that the copy should read something like, "Your profile can be found at {{url}}"?